File: runtimes/java-8.0/src/main/java/io/openruntimes/java/RuntimeOutput.java
package io.openruntimes.java;

import java.util.Map;

public class RuntimeOutput {
    private final Object body;
    private final int statusCode;
    private final Map<String, String> headers;

    public RuntimeOutput(
            Object body,
            int statusCode,
            Map<String, String> headers
    ) {
        this.body = body;
        this.statusCode = statusCode;
        this.headers = headers;
    }

    public Object getBody() {
        return this.body;
    }

    public int getStatusCode() {
        return this.statusCode;
    }

    public Map<String, String> getHeaders() {
        return this.headers;
    }
}

File: runtimes/dotnet/versions/latest/src/RuntimeOutput.cs
namespace DotNetRuntime
{
    public class RuntimeOutput
    {
        public byte[] Body { get; set; }
        public int StatusCode { get; set; }
        public Dictionary<string, string> Headers { get; set; }

        public RuntimeOutput(byte[] body, int statusCode, Dictionary<string, string> headers)
        {
            Body = body;
            StatusCode = statusCode;
            Headers = headers;
        }
    }
}
